script.js

```diff
@@ -0,0 +1,95 @@
+let defaultResult = 0;
+
+let currentResult = defaultResult;
+
+let logEntries = [];
+
+function getUserInput() {
+  return parseInt(userInput.value);
+}
+
+// the description & result output of calculation
+
+function createOutput(numberBeforeCalc, operator, calcNumber) {
+  const calcDescription = `${numberBeforeCalc} ${operator} ${calcNumber}`;
+  outputResult(currentResult, calcDescription);
+}
+
+// object inside array
+
+function writeToLog(
+  operationIdentifier,
+  prevResult,
+  numberEnteredByUser,
+  finalResult
+) {
+  const logEntry = {
+    operation: operationIdentifier,
+    PrevResult: prevResult,
```

Comment:
camelCase ?

Reply:
will update soon

Reply:
updated, check it @faran4engg

Reply:
logEntry was correct. My comment was  intended for `PrevResult` . It should be `prevResult`
@Rehan-Khan-Engineer

Reply:
Done @faran4engg